simple_imports/README.md

```diff
@@ -0,0 +1,41 @@
+# Simple Imports
+
+This folder contains configs for imports using [Simple Stats
+Importer](https://github.com/datacommonsorg/import/edit/master/simple/README.md).
+
+## Import config
+
+Each import has a sub-folder with a config file `<import-name>/simple_import.json`.
```

Comment:
Perhaps add a note that a given organization can arrange their imports as sub-folders under a single parent?

Reply:
Done